recipes/filelock/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "filelock" %}
+{% set version = "2.0.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+   fn: {{ name }}-{{ version }}.tar.gz
+   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+   sha256: 9209380b587f60c7533c2f204e468ee9dea898a79f437de5e5e3d8618e4f1e23
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - filelock
+
+about:
+  home: https://github.com/benediktschmitt/py-filelock
+  license: Unlicense
```

Comment:
Based on the [license]( https://github.com/benediktschmitt/py-filelock/blob/fde6c13718dd98ca50a9e3378f972706035b57c6/LICENSE.rst ), I would just say "Public Domain". This is actually what we do with sqlite, as well.